examples/nextjs/README.md

```diff
@@ -1,3 +1,36 @@
 # example: nextjs
 
-This example is built based on the instructions from [Learn Next.js](https://nextjs.org/learn).
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+```shell
+npx create-next-app@latest nextjs --js --tailwind --no-eslint --app --src-dir --turbopack --no-import-alias --use-npm
+```
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `app/page.js`. The page auto-updates as you edit the file.
+
+This project uses [`next/font`](https://nextjs.org/docs/app/building-your-application/optimizing/fonts) to automatically optimize and load [Geist](https://vercel.com/font), a new font family for Vercel.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
```

Comment:
@MikeMcC399 I wasn't sure if this was autogenerated content as well, but if not - could we link to Cypress' Next.js docs? https://docs.cypress.io/app/component-testing/react/overview#Nextjs

Reply:
The README is indeed autogenerated content although I did add a few lines at the beginning to document exactly how I created it.

This example covers only E2E tests, so linking to Cypress documentation for component testing of Next.js would likely be confusing unless there was some additional explanation to go with the link.

Reply:
The current README is short and no longer valid as the contents of https://nextjs.org/learn have completely changed.

![image](https://github.com/user-attachments/assets/8b00b52b-58e0-4b69-850d-9e58b4e2beb7)